### DISCUSSION TITLE
systemd-homed might be very useful for ltsp

### DISCUSSION THREAD

**Discussion by enaut** [ltsp/ltsp#176] [Q&A]
There is a new thing popping up in systemd that seems to be the perfect system for ltsp to share home directories.
It will probably take a Version or two of Ubuntu until it is available... so this is rather a longterm-ticket. But the talk and description show great potential. It also might make a lot of the things currently done rather complicated much easier. So it might be worthwhile having this in mind while implementing things like guest users and so on.

Sources:
  * description: https://github.com/poettering/systemd/blob/homed/docs/HOME_DIRECTORY.md
  * git: https://github.com/poettering/systemd/blob/homed
  * talk: (English): https://cfp.all-systems-go.io/ASG2019/talk/VSQRXA/

**Comment by alkisg** [comment]
Hi enaut,

yes I read about it and it's interesting; but I don't think it will affect the LTSP codebase in the next couple of years, and it's possible that it'll be transparently handled by pam_systemd anyway.

So I'm transferring this to the community issues; we can put it back to issues if specific action is needed.
